src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -99,7 +99,13 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
 
         internal override IEnumerable<AssociativeNode> BuildAst(List<AssociativeNode> inputAstNodes, CompilationContext context)
         {
-            var rhs = AstFactory.BuildStringNode(Value);
+            string value = Value;
+            if (context == CompilationContext.NodeToCode)
+            {
+                value = value.Replace(@"\", @"\\")
+                    .Replace("\"", "\\\"");
+            }
+            var rhs = AstFactory.BuildStringNode(value);
```

Comment:
I thought about this approach but I think this is affecting a lot of different behaviour. Example, it changes the node previews, watch node previews etc. for String node, which is why I refrained from suggesting this.